### PR TITLE
fixes for embedded DR snapshots

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx [hunk 1]
```diff
@@ -287,7 +288,7 @@ class SnapshotSchedule extends Component {
                       className="u-cursor--pointer"
                       id="autoEnabled"
                       checked={this.state.autoEnabled}
-                      disabled={(!isAppConfig && isKurlEnabled)}
+                      disabled={false}
```
Comment (salahalsaleh) [inline kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx:291]: you can just remove this line

File: kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx [hunk 2]
```diff
@@ -312,7 +313,7 @@ class SnapshotSchedule extends Component {
                         getOptionValue={(schedule) => schedule.label}
                         value={selectedSchedule}
                         onChange={this.handleScheduleChange}
-                        isDisabled={(!isAppConfig && isKurlEnabled)}
+                        isDisabled={false}
```
Comment (salahalsaleh) [inline kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx:316]: ditto

File: kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx [hunk 3]
```diff
@@ -378,8 +376,8 @@ class SnapshotSchedule extends Component {
           err="Failed to get snapshot schedule settings"
           loading={loadingConfig}
         />
-        {!isAppConfig && isKurlEnabled &&
-          <GettingStartedSnapshots isSettingsPage={true} />
+        {!isAppConfig && !isSettingsPage &&
+          <GettingStartedSnapshots isVeleroInstalled={true} />
```
Comment (salahalsaleh) [inline kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx:380]: are you sure?

Comment (divolgin) [reply kotsadm/web/src/components/snapshots/SnapshotSchedule.jsx:380]: yep, `isSettingsPage` isn't used inside of GettingStartedSnapshots.  At any rate, no modal should be blocking settings pages.

**Review by salahalsaleh — APPROVED** [review]

**Pull request merged**